
From: Jesper Juhl <juhl-lkml@dif.dk>

drivers/atm/ambassador.c: In function `do_pci_device':
drivers/atm/ambassador.c:2295: warning: unsigned int format, long unsigned int arg (arg 2)
drivers/atm/ambassador.c:2295: warning: unsigned int format, long unsigned int arg (arg 2)

Signed-off-by: Jesper Juhl <juhl-lkml@dif.dk>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/atm/ambassador.c |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff -puN drivers/atm/ambassador.c~drivers-atm-ambassador.c-do_pci_device-printk-warning-fix drivers/atm/ambassador.c
--- 25/drivers/atm/ambassador.c~drivers-atm-ambassador.c-do_pci_device-printk-warning-fix	2004-10-03 16:56:39.139803376 -0700
+++ 25-akpm/drivers/atm/ambassador.c	2004-10-03 16:56:39.144802616 -0700
@@ -2293,7 +2293,7 @@ static int __init do_pci_device(struct p
 	u8 irq = pci_dev->irq;
 
 	PRINTD (DBG_INFO, "found Madge ATM adapter (amb) at"
-		" IO %x, IRQ %u, MEM %p", pci_resource_start(pci_dev, 1),
+		" IO %lx, IRQ %u, MEM %p", pci_resource_start(pci_dev, 1),
 		irq, bus_to_virt(pci_resource_start(pci_dev, 0)));
 
 	// check IO region
_
